build.gradle

```diff
@@ -708,7 +708,7 @@ task fakePublish {
 
 }
 
-if (project.hasProperty('mavenUser') && project.hasProperty('jcecertAlias')) {
```

Comment:
don't we still need the mavenUser to publish?

Reply:
We're using [environment variables](https://github.com/corretto/amazon-corretto-crypto-provider/blob/main/build.gradle#L770-L771) that specify a user token and password for publishing. The `gradle.properties` file that we'll use for releases will not have `mavenUser` defined in it anymore.